packages/babel-parser/test/fixtures/experimental/export-extensions/default-from-identifier-escaped/input.js

```diff
@@ -0,0 +1 @@
+export default \u{66}rom;
```

Comment:
🤦‍♂️IMO it should be addressed in the export-default-from proposal that while `export default from` is banned,  `export default \u{66}rom` is still valid.